packages/agreement-process/src/routers/AgreementRouter.ts

```diff
@@ -93,16 +94,21 @@ const agreementRouter = (
     "/agreements/:agreementId/submit",
     authorizationMiddleware([ADMIN_ROLE]),
     async (req, res) => {
+      const ctx = fromAppContext(req.ctx);
```

Comment:
About this: if not already done, I would open an improvement on Jira, to understand if we can avoid somehow using `fromAppContext` everywhere and make it work by just adding the logger to the `req.ctx` and make TS like it 🙏

Reply:
https://pagopa.atlassian.net/browse/IMN-513